.github/workflows/test.yml

```diff
@@ -20,6 +20,15 @@ jobs:
       CC: ${{ matrix.compiler }}
     steps:
       - uses: actions/checkout@v2
+      - name: install dependencies
```

Comment:
This would require every Workflow that uses this Action to include this snippet, which is undesirable.

Reply:
I would like to use this action to install DPDK even in distributions where apt/apt-get is not the default packet manager (e.g. yum or dnf). 
That is why I thought it might be beneficial to move installation of dependecies outside of the action even for the price of undesirable effect you mentioned.

Reply:
If you want other OS support, still  write the `apt-get` command in the script, inside a conditional so that it only runs on Ubuntu and Debian.
Workflow using other OS can manually install those dependencies.

Reply:
I have done it as you wish. Although I have implemented it, I am still not convinced it is a safe path to take. I think some inconsistencies across distrubitions may occur in the future and those might be hard to solve later on.

Reply:
There would be inconsistency for sure. Thus, only include steps for Ubuntu that is the only OS supported on GitHub runners. For all other OS, dependencies should still be installed in the workflow.